src/test/groovy/org/prebid/server/functional/tests/privacy/GppSetuidSpec.groovy

```diff
@@ -0,0 +1,43 @@
+package org.prebid.server.functional.tests.privacy
+
+import org.prebid.server.functional.model.UidsCookie
+import org.prebid.server.functional.model.request.setuid.SetuidRequest
+import org.prebid.server.functional.service.PrebidServerException
+import org.prebid.server.functional.util.PBSUtils
+
+import static org.prebid.server.functional.model.bidder.BidderName.GENERIC
+
+class GppSetuidSpec extends PrivacyBaseSpec {
+
+    def "PBS setUid shouldn't reject request when gpp is invalid"() {
+        given: "Set uid request with invalid GPP"
+        def setUidRequest = SetuidRequest.defaultSetuidRequest.tap {
+            it.gpp = "Invalid_GPP_Consent_String"
+            it.gppSid = null
```

Comment:
Please parametrize the SID. You'll want to also check some valid values as some of the logic is tied to a SID presence.